content/en/docs/instrumentation/java/automatic/spring-boot.md

```diff
@@ -6,24 +6,32 @@ description: Spring instrumentation for OpenTelemetry Java
 cSpell:ignore: autoconfigure datasource logback springboot springframework
 ---
 
-You can use the [OpenTelemetry Java agent](..) with byte code instrumentation to
-automatically instrument a [Spring Boot](https://spring.io/projects/spring-boot)
-application; or you can also use the OpenTelemetry [Spring Boot starter] to
-instrument your application.
+## How to instrument a Spring Boot application
```

Comment:
I feel the title here is redundant. You can eliminate it and use the doc title (or, better yet, rewrite the description).